packages/dev/core/src/FlowGraph/Blocks/Execution/flowGraphTimerBlock.ts

```diff
@@ -0,0 +1,67 @@
+import type { FlowGraphDataConnection } from "../../flowGraphDataConnection";
+import type { FlowGraphSignalConnection } from "../../flowGraphSignalConnection";
+import { AdvancedTimer } from "../../../Misc/timer";
+import type { FlowGraphContext } from "../../flowGraphContext";
+import { FlowGraphAsyncExecutionBlock } from "../../flowGraphAsyncExecutionBlock";
+import { RichTypeNumber } from "../../flowGraphRichTypes";
+
+/**
+ * @experimental
+ * Block that can execute an action immediately and another after a delay.

```

Comment:
what is the meaning of immediately here? Is this really two different tasks, or is it the sync vs async flow?

Reply:
the block has two different output flows, one that is activated synchronically and another asynchronically

Reply:
i think the codedoc might be a little misleading. It will resolve right after triggering the timeout, but will trigger the actual reason for this block after N ms